docs/hash/Hashtables.md

```diff
@@ -0,0 +1,129 @@
+---
+id: Hash-tables-Dsa

```

Comment:
replace `id: Hash-tables-Dsa` with `id: hash-tables-dsa` for good practice

Reply:
only changing the id and renaming the files is enough or should I check on the code too?

Reply:
You need to sync our latest code. I've resolved the error, so first, sync with `git pull origin main`, push the code to your branch, and then resolve any remaining errors.

Reply:
Done can you check now?